content/blog/2022/11/airgap.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Air gap
+authors:
+- Raul Cabello Martin
+date: 2022-11-28
+---
+
+We are glad to announce that Kubewarden can now be installed in air-gapped environments! For that, you will need a 
```

Comment:
```suggestion
We are glad to announce that deploying Kubewarden in air-gapped environments has been simplified and documented! For that, you will need a 
```
(trying to make the point to users that we have made it just even simpler, and transmit that we already had a good stack, instead of an intermingled one)